draft-ietf-httpbis-http2bis.xml

```diff
@@ -619,28 +620,28 @@
         </t>
       </section>
       <section anchor="HeaderBlock">
-        <name>Header Compression and Decompression</name>
+        <name>Field Section Compression and Decompression</name>
         <t>
-          Just as in HTTP/1, a header field in HTTP/2 is a name with one or more associated values.
-          Header fields are used within HTTP request and response messages as well as in server push operations
+          A field in HTTP is a name with one or more associated values.
+          Field lines carry each value and are used within HTTP/2 request and response messages as well as in server push operations
```

Comment:
HPACK compresses both fields and control data and applies to "field lines", but control data is not a field?  This might need more explanation.

Reply:
Yeah I think this specific change doesn't read right to me. Is "field block" really right? That feels very strange.

Reply:
I've given this another go, with a more comprehensive rewrite of the first few paragraphs of this section.  Let me know if that reads better.

Reply:
I do think it's better, yeah.